frontend/src/pages/Contributions.jsx

```
@@ -0,0 +1,162 @@
+            )}
+          </div>
+        </div>
+
+        <div className="video-grid">
+          <div className="video-card">
+            <div className="video-thumbnail">
+              <h3>learn {activeTag}</h3>
+              <button 
+                onClick={() => handleViewClick('learn-algorithm')}
+                className="view-btn"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)